common/src/main/java/com/microsoft/identity/common/internal/controllers/CommandDispatcher.java

```diff
@@ -341,6 +350,14 @@ public void onReceive(Context context, Intent intent) {
                     sCommand = null;
                     localBroadcastManager.unregisterReceiver(resultReceiver);
 
+                    // set correlation id on Local Authentication Result
+                    if (commandResult.getResult() != null &&
+                            commandResult.getResult() instanceof LocalAuthenticationResult) {
+                        final LocalAuthenticationResult localAuthenticationResult =
+                                (LocalAuthenticationResult) commandResult.getResult();
+                        localAuthenticationResult.setCorrelationId(correlationId);
+                    }
+
```

Comment:
should this become a method?

Reply:
@AdamBJohnsonx Addressed here: https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/925/commits/21e14b3679de09f10658a2f08142f1ca41182627